tests/static_tests/iterator_t.cpp

```diff
@@ -52,17 +52,6 @@ concept LegacyInputIterator =
         requires std::signed_integral<typename std::incrementable_traits<Iter>::difference_type>;
     };
 
-template<class Iter, class Value>
-concept can_iterate_mapped = requires(Iter it) {
-    requires LegacyInputIterator<Iter>;
-    // explicit check of the sentinel role, since `std::ranges::borrowed_range` in `can_view_mapped`
-    // would not tell us why exactly the end iterator cannot be a sentinel
-    requires std::sentinel_for<Iter, Iter>;
-    { *it } -> std::same_as<Value&>;
-    // note: should actually be only present for contiguous iterators
-    { it.operator->() } -> std::same_as<Value*>;
```

Comment:
Just remove this line from the concept check, not the whole concept!

Reply:
done